client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -1,9 +1,14 @@
 import { shield, trendingUp, chartBar } from '@wordpress/icons';
 import { translate, useTranslate } from 'i18n-calypso';
+import { ReactNode } from 'react';
 import customerImageAjitBohra from 'calypso/assets/images/migrations/customer-testimonials/ajit-bohra.jpg';
 import customerImageAntonyAgnel from 'calypso/assets/images/migrations/customer-testimonials/antony-agnel.jpg';
 import customerImageChrisCoyier from 'calypso/assets/images/migrations/customer-testimonials/chris-coyier.jpg';
 import customerImageEmmaLucasCopley from 'calypso/assets/images/migrations/customer-testimonials/emma-lucas-copley.jpg';
+import { UseGetUpgradePlanSiteMetrics } from './hooks/use-get-upgrade-plan-site-metrics';
+import wordpressCwvtechReportJson from './wordpress-cwvtech-report.json';
+
+export const upgradePlanSiteMetricsLcpThreshold = 2500;
```

Comment:
Let's add a comment to explain this here, paraphrasing this:

```
Threshold for “Good”: A webpage is considered to have a “good” Core Web Vital score if it meets the following thresholds for each metric:
LCP: Loads within 2.5 seconds of navigation start.
```

Reply:
Added comment for this constant